### DISCUSSION TITLE
Is someone working on a BlueRetro internal PCB for the N64?

### DISCUSSION THREAD

**Discussion by Roy-Meijer** [darthcloud/BlueRetro#822] [Ideas]
Hello,

I designed a PCB for the N64 which can be used to reset the console using your controller. I was thinking about selling this PCB, but then someone told me I can try to make a PCB which works with BlueRetro instead. I think this is a good idea because then it can add a lot more functionality than only resetting the console and it seems all software and schematics are already available in this repository. The only problem is that I don't know if someone else is already working on such a PCB. Do you know if someone else is already working on a BlueRetro internal PCB for the N64?

**Comment by darthcloud** [comment]
tharathiel over The New Retro discord did. Not sure if its puplic yet.

**Comment by darthcloud** [comment]
https://github.com/TharathielCB/BR4N64

Its now public
